config.js

```diff
@@ -119,6 +119,16 @@ config.iota = {
         port: 4061
     },
 
+    authentication: {
```

Comment:
Same as https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r258419452

Reply:
Awaiting clarification - see https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r258528839

Reply:
Fixed (maybe): 51deb3c

I have **annotated**  the section I've added to the `config.js` and then commented the thing out - same as the `iotmanager`. This satisfies my own needs of being able to use the `config.js` as **documentation**.  But it doesn't saisfy the request made by @fgalan to:

> let's keep it simple are avoid changing the `config.js` file.

Since I've changed the file. I assume a _commented out_  config should still work since:

> it seems the `env var` processing logic is able to create what can be missing from `config.js`.

So is this Fixed, or should I just delete the `authentication` section altogether?

Reply:
Included the section as a commented block as you did in 51deb3c is a good trade off between "keeping it simple and avoid chaning the config.js" and inline documentation in the config.js itself. So, it's fine with me and I'll suggest to do the same for the "twin" PR in IOTA-UL.